cpg_infra/billing_aggregator/aggregate/seqr.py

```diff
@@ -425,12 +425,17 @@ def generator() -> Generator[dict, None, None]:
             _obj_param_map = seqr_wide_param_map
             if 'dataset' in labels:
                 # specific override where 'dataset' is specified in GCP resource
+                # MiloH question: labels seems to have format {'key': 'KEY1, 'value': 'VAL'}
+                # so how does this work?
                 _obj_param_map = {labels['dataset']: (1.0, 1)}
```

Comment:
I think it never works 😓 
Can you do the format earlier so this block works?

Reply:
As suggested, reformatted labels prior this check.